crates/sdk/libs/near/src/types.rs

```diff
@@ -0,0 +1,31 @@
+use serde_with::base64::Base64;
+use serde_with::serde_as;
+
+pub type BlockHeight = u64;
+pub type BlockHash = calimero_primitives::hash::Hash;
+pub type AccountId = near_account_id::AccountId;
+pub type StorageUsage = u64;
+pub type Nonce = u64;
+pub type Balance = u128;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[serde(untagged)]
+pub enum BlockId {
+    Height(BlockHeight),
+    Hash(BlockHash),
+}
+
+#[serde_as]
+#[derive(serde::Deserialize, Clone, Debug)]
+#[serde(transparent)]
+pub struct StoreValue(#[serde_as(as = "Base64")] pub Vec<u8>);
+
+#[serde_as]
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug)]
+#[serde(transparent)]
+pub struct StoreKey(#[serde_as(as = "Base64")] pub Vec<u8>);
+
+#[serde_as]
+#[derive(serde::Serialize, Clone, Debug)]
+#[serde(transparent)]
+pub struct FunctionArgs(#[serde_as(as = "Base64")] Vec<u8>);
```

Comment:
why aren't these with view?

Reply:
The folder structure is based on near-premitives crate.

Reply:
Uh huh.. feel free to organize things in your own way, they would've done what they did for reasons beyond the scope of this PR

Like for example, these structures could be reused when submitting transactions, which we don't do right now.

Reply:
ok ok :)

Reply:
Currently, everything is `query` related so that we can move everything to `query` module.